### PR TITLE
Fix "F403", # `from python_terraform import *` used; unable to detect undefined names

### PR DESCRIPTION
**Pull request by jiridanek**

### CODE REVIEW

File: ods_ci/utils/scripts/terraform/openstack/provision.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 import shutil
 import sys
 
-from python_terraform import *
+from python_terraform import IsFlagged, IsNotFlagged, Terraform
```
Comment (jiridanek) [inline ods_ci/utils/scripts/terraform/openstack/provision.py:6]: Do we actually still use this provision.py for anything? I wanted to test this in Jenkins and I can't find anything that runs this.

**Comment by jiridanek** [comment]
@aloganat confirmed this script is not currently used in CI, so I checked it by hand

1. create `~/.config/openstack/clouds.yaml`
2. execute `poetry run python /home/jdanek/repos/ods-ci/ods_ci/utils/scripts/terraform/openstack/provision.py create_instance --cloud_name=rhos-d --vm_name=jdanek --vm_user=jdanek --vm_private_key=cci --key_pair=cci --network_name=provider_net_cci_9`
3. check in webui that vm got created
4. execute `poetry run python /home/jdanek/repos/ods-ci/ods_ci/utils/scripts/terraform/openstack/provision.py delete_instance --cloud_name=rhos-d --vm_name=jdanek --key_pair=cci --network_name=provider_net_cci_9`
5. check vm got deleted

marking the PR as verified

File: ods_ci/utils/scripts/terraform/openstack/provision.py (hunk 1 above)
Comment (jiridanek) [reply ods_ci/utils/scripts/terraform/openstack/provision.py:6]: we don't use it now, but we want to keep it

**Review by aloganat — APPROVED** [review]
lgtm

**Review by jstourac — APPROVED** [review]

**Pull request merged**